recipes/pyflyby/meta.yaml

```diff
@@ -0,0 +1,50 @@
+{% set name = "pyflyby" %}
+{% set version = "1.6.2" %}
+
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/{{ name }}-{{ version }}.tar.gz
+  sha256: 5bebd9fe0f40d809bbf029e3d0d1bfa7bfe2a94379f5a8f383cb85d94fb9a88c
+
+build:
+  skip: true  # [win]
```

Comment:
`noarch: python` would be much better, even if it doesn't work on windows

Reply:
How do we skip it on Windows? See the previous builds for the failure. It doesn't even import.

Reply:
It doesn't need to build on windows. It only needs to build on linux if is noarch.

Reply:
So we don't need to skip on windows

Reply:
I guess I don't understand how to make that work then. Does staged recipes always build on Windows even if it's noarch? I had the recipe as noarch: python but it was building on all three platforms.

Reply:
yes, staged-recipes always builds on all three platforms, so windows is an allowed failure in this case

Reply:
OK, I reverted the noarch removal commits.